docs: update canvas buttons, executions, and projects wording (#3514)
n8n-io/n8n-docs: docs/courses/level-one/chapter-1.md

The panel contains the following sections:

- **Overview**: Contains all the workflows and credentials you have access to. During this course, create new workflows here.

> - **Projects**: (Community edition shows this as personal) Projects group workflows and credentials. You can assign [roles](/user-management/rbac/role-types.md) to users in a project to control what they can do in a project. A **Personal** project is available by default.

Reviewer: ```suggestion
- **Projects**:  Projects group workflows and credentials. You can assign [roles](/user-management/rbac/role-types.md) to users in a project to control what they can do in a project. A **Personal** project is available by default. Projects aren't available on the Community edition.
```
